### ISSUE TITLE
Do you have pretrained model?

### ISSUE THREAD

**Issue by tqvinhcs** [microsoft/View-Adaptive-Neural-Networks-for-Skeleton-based-Human-Action-Recognition#6]
Hi,
I am looking for pre-trained models on NTU CS and CV dataset for the va-cnn.
It would be great if you can provide these pre-trained models.
Also, do you have code for training on the SYS3DHOI dataset for va-cnn?

Thank you

**Comment by shuidongliu** [comment]
You can download the pre-trained model through https://drive.google.com/open?id=18QRqGvJygU5ydYxTpEfI6jANR72C_2rc.

**Comment by preksha12** [comment]
<img width="501" alt="Screenshot 2020-04-11 at 1 57 13 PM" src="https://user-images.githubusercontent.com/9927293/79039097-86194700-7bfc-11ea-975c-65a7b8272a2d.png">
I am getting problem in result as mentioned in paper accuracy is 93 but I am getting only 73

**Comment by shuidongliu** [comment]
Hi, there is something wrong with the README.md file. When testing the
model, you should shut down the data augmentation by "python va-cnn.py
--model VA --aug 0 --train 0"

Best,
Pengfei

preksha12 <notifications@github.com> 于2020年4月11日周六 下午4:29写道：

**Comment by shuidongliu** [comment]
Hi, there is something wrong with the README.md file. When testing the model, you should shut down the data augmentation by "python va-cnn.py --model VA --aug 0 --train 0"

**Comment by preksha12** [comment]
Hello,
Thank you for your response but after calling above program model run for 2 times with accuracy with model accuracy 41 and 83.85 and average accuracy of 62

**Comment by shuidongliu** [comment]
Hello,
I just download the modeles and test them with the released code, the
results are as follows:
[image: image.png]

According to my experience, the testing results will not change with the
released code. Do you change the code?

preksha12 <notifications@github.com> 于2020年4月11日周六 下午8:44写道：

**Comment by shuidongliu** [comment]
Hello,
I just download the modeles and test them with the released code, the results are as follows:

Train on 38086 samples, validate on 2005 samples
Model Accuracy:88.74
Train on 35763 samples, validate on 1883 samples
Model Accuracy:94.28
[88.74, 94.28]
ave: 91.50999999999999

According to my experience, the testing results will not change with the released code. Do you change the code?

**Comment by preksha12** [comment]
Thank you for your quick reply. Actually as per the below link of issue I was getting error
https://github.com/microsoft/View-Adaptive-Neural-Networks-for-Skeleton-based-Human-Action-Recognition/issues/7#issue-595366294

Due to above error I have made following changes:-
In va-cnn.py: I changed worker=0 in place of default=8 in line 43
In data_cnn.py: due to error in scipy.misc imresize function I have used rgb_ske=np.resize(rgb_ske,(224,224,3)) in line number 94.

Could you please guide me how to resolve these issue without changing code as I am using python3.6.

**Comment by shuidongliu** [comment]
It may caused by the Scipy as mentioned in https://stackoverflow.com/questions/56204985/how-to-fix-scipy-misc-has-no-attribute-imresize/56205147.
There is no problem when I use Python 3.6. Therefore, I check the version of Scipy of mine. And the information is as follows:
Python 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 18:10:19)
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import scipy
>>> scipy.__version__
'1.0.0'
>>>

**Comment by preksha12** [comment]
I have installed scipy 1.1.0 and now it stated running but there I have few more doubts:
1) How can I make this model to run for GPU?
2) In va-rnn I am getting attached error. 
In va-rnn code I have updated tensor flow as backend in place of theano:
<img width="1280" alt="Screenshot 2020-04-13 at 3 46 09 PM" src="https://user-images.githubusercontent.com/9927293/79113177-ed223180-7d9d-11ea-921d-cb7933b181b6.png">


it will be great if it is possible for you to give email address for communication.

**Comment by shuidongliu** [comment]
For va-cnn, it should run for GPU by default.  Could you give me more information?
For va-rnn, I tried tensorflow before, but the results are poor and the training speed is slow. Therefore, I suggest to use theano. 
You can communicate with me through zpengfei@stu.xjtu.edu.cn

**Comment by Anupama-898** [comment]
can you please provide the pretrained model for VA-RNN also.

**Comment by shuidongliu** [comment]
> can you please provide the pretrained model for VA-RNN also.

You can download both models for VA-RNN and VA-CNN through https://drive.google.com/open?id=12PfMRK2rSg0ndLbTVw-GelRrb2TVxazg

**Comment by Anupama-898** [comment]
Thakyou for the quick response. Is is possible to view the 3D skeletal poses from view adaption subnetwork as shown in paper?
